### DISCUSSION TITLE
How to unblock ANYTHING!

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#311] [Announcements]
**YouTube video: https://www.youtube.com/watch?v=6CpsZPWTqxk**

**Comment by bartholemuew** [comment]
real

**Reply by akabutnicer** [reply]
First of all, it costs money, second of all WHO has a terminal available if they did, they wouldn't need to be here! And why not just put replit?

**Reply by piplup7575** [reply]
> First of all, it costs money, second of all WHO has a terminal available if they did, they wouldn't need to be here! And why not just put replit?

Hey @James1234121 ! You can actually use this for free for several months if you use Echo's partner link, [https://linode.gvw92c.net/3kh0](https://linode.gvw92c.net/3kh0)

**Reply by Aka-but-nice** [reply]
Also James1234121 is my alt account

**Reply by Christianrunbruh** [reply]
heres my card number to get started with free trial 4361 2710 4242 5803 security code:065 put in as debit card

**Reply by Aka-but-nice** [reply]
Kid that's not even how it works, you need the date, and we know it is fake

**Reply by bartholemuew** [reply]
> First of all, it costs money, second of all WHO has a terminal available if they did, they wouldn't need to be here! And why not just put replit?

why u complaining to me i just said real

**Reply by Aka-but-nice** [reply]
? I'm talking to @Christianrunbruh

**Reply by Christianrunbruh** [reply]
ita a spendwell card and it has no money so i gave it out for free linode trial

**Reply by Christianrunbruh** [reply]
poof that it is real ill send a vid

**Reply by Christianrunbruh** [reply]
and heres the exp 30/12

**Reply by ineedyoutubeworkingonschoolcromebook** [reply]
uhhhh

**Comment by ehunt34** [comment]
Would've been helpful if it wasn't paid and if we had a terminal on this damn "Gentoo but worse" computer. If we had a terminal we could run this native. What is even the point?

**Comment by Aka-but-nice** [comment]
THATS WHAT IM SAYIN!

**Reply by erasereater** [reply]
ovdih beee
![Screenshot 2023-01-24 7 47 09 AM](https://user-images.githubusercontent.com/123754393/215128778-3acf0822-0667-4c98-98e2-83d0c09c2292.png)

**Reply by Christianrunbruh** [reply]
goofy ahh

**Comment by Alex2Mayhem** [comment]
> **YouTube video: https://www.youtube.com/watch?v=6CpsZPWTqxk**

First. We don't have windows. Most of us has ChromeOS. Second, we aren't paying for something when possibly it could get patched. Third, linode is blocked for most.

**Reply by Aka-but-nice** [reply]
well it's not for me, but it's not free, and I actually do have access to a terminal but ssh is disabled by my admin so it won't allow me to run any commands when I go to it it just says " Loading pnacl program..." forever and never loads!

**Reply by 3kh0** [reply]
it does not cost money if you use my link, you get a free 5 month trial, see if you like it

**Reply by Alex2Mayhem** [reply]
> it does not cost money if you use my link, you get a free 5 month trial, see if you like it

The website is blocked.

**Reply by Aka-but-nice** [reply]
@3kh0 does it use a credit card

**Reply by 3kh0** [reply]
yes

**Reply by md4r2** [reply]
@3kh0 Is it possible to do this on replit?

**Reply by 3kh0** [reply]
@md4r2 no

**Reply by Aka-but-nice** [reply]
so tf the point of this, all you did was put a bunch of random shit together, and made it work.

**Reply by Christianrunbruh** [reply]
heres my card number to get started with free trial 4361 2710 4242 5803 security code:065 put in as debit card

**Reply by Christianrunbruh** [reply]
it is real

**Reply by Aka-but-nice** [reply]
> heres my card number to get started with free trial 4361 2710 4242 5803 security code:065 put in as debit card

That's not how credit cards work. I already have a card.

**Reply by Christianrunbruh** [reply]
i have DEBIT card
exp 30/12

**Comment by Aka-but-nice** [comment]
Wait but what about the terminal that still ain't fixed.

**Comment by erasereater** [comment]
huh
